tests/JustSaying.Extensions.DependencyInjection.StructureMap.Tests/StructureMapTests.cs

```diff
@@ -32,8 +34,10 @@ public async Task Can_Create_Messaging_Bus_Fluently_For_A_Queue()
                     (builder) =>
                     {
                         builder.Client((options) =>
-                                options.WithBasicCredentials("accessKey", "secretKey")
-                                    .WithServiceUri(TestEnvironment.SimulatorUrl))
+                                    options.WithClientFactory(() => new LocalAwsClientFactory(InMemoryAwsBus))
+                                // options.WithBasicCredentials("accessKey", "secretKey")
+                                //     .WithServiceUri(TestEnvironment.SimulatorUrl)
```

Comment:
My intention was to leave this commented out, then in a separate PR re-add these back in (with Aspire too) so that they run again with CI.
Feels a bit cowboy-ish, but I don't want to add more tasks to this PR, and I don't want to erase these bits as they will act as markers for the follow up task.